bindata/manifests/switchdev-config/files/switchdev-configuration-before-NM.sh.yaml

```diff
@@ -67,12 +67,5 @@ contents:
 
         # turn hw-tc-offload on
         /usr/sbin/ethtool -K ${name} hw-tc-offload on
-
```

Comment:
not related to this PR, but we could probably save some time during boot if we set `sriov_drivers_autoprobe` to 0
then there is no need to unload drivers in line 57

1. echo 0 > /sys/bus/pci/devices/PCI_DBDF_OF_PF/sriov_drivers_autoprobe
2. echo XXX > /sys/bus/pci/devices/PCI_DBDF_OF_PF/sriov_numvfs
3. Restore drivers autoprobe (just for correctness) 
    echo 1 > /sys/bus/pci/devices/PCI_DBDF_OF_PF/sriov_drivers_autoprobe

if this sounds like something we like, i can open an enhancement on it

Reply:
Sounds good to me. Can `echo 1 > /sys/bus/pci/devices/PCI_DBDF_OF_PF/sriov_drivers_autoprobe` load the driver to VFs as we do in line 75?

Reply:
need to re-enable sriov_drivers_autoprobe before binding driver in L75

Reply:
done #214